Rules.md

```diff
@@ -176,59 +176,60 @@ Sea-sea movement is not possible for a unit if any land region at the strait is
 
 ## Battles
 
-When a move proceeds into an enemy region, a battle is initiated with all moved units.
+When a move proceeds into an enemy region, a battle is initiated between all units moved into the
+region versus the enemy units already in that region.
 
 ### Battle mechanic
 
-A battle has one or more rounds. Every round has following parts:
+A battle has the following phases:
 
 1. **Artillery strike:**
-    * Every attacking artillery kills 1 unit (\*)
+    * Every attacking artillery kills 1 unit (\*).
 1. **Rolling dice:**
-    * Attacker gets a dice for every attacking **infantry unit**
-    * Defender gets a dice for every defending **unit** (infantry + artillery)
-    * Every attacker roll of:
-        * **3-4:** Forces an enemy unit to retreat
-        * **5-6:** Kills an enemy unit
-    * Every defender roll of:
-        * **1-4:** Forces an enemy unit to retreat
-        * **5-6:** Kills an enemy unit
-
-   Kills are resolved before forced retreat.
-1. **Attacker retreat:** The attacker chooses their forced retreating units plus any additional units
-   they want to retreat. All these units move back to the region where the move started.
-
-   If no attacking units are left, the battle is instantly over and any defending units that would
-   have been forced to retreat, don't retreat but remain in the region.
-1. **Defender retreat:** The defender chooses their forced retreating units plus any additional units
-   they want to retreat. All these units move to a single adjacent land region that is either owned by
-   the defender or unoccupied, but not the region where the attacker's move started. Retreating units
-   in subsequent battle rounds need to retreat to this same area.
+    * The attacker gets a dice for every attacking **infantry unit**
+    * The defender gets a dice for every defending **unit** (infantry + artillery)
+    * Every attacker roll of
+        * **1-2** equals 1 military power,
+        * **3-4** equals 2 military power,
+        * **5-6** equals 2 military power and kills an enemy unit. (\*)
+    * Every defender roll of
+        * **1-4** equals 2 military power,
+        * **5-6** equals 2 military power and kills an enemy unit. (\*)
+   The attacker gets an additional 2 military power for each of his surviving artillery.
```

Comment:
Suggestion: Remove this and instead, artilleries get a separate dice that has the same numbers as infantries, but without kills.